client/.dockerignore

```diff
@@ -10,3 +10,4 @@
 !src
 !public
 !.env*
+ !docker-entrypoint.sh
```

Comment:
Same. Can we invert this ? Include everything but the commonly excluded files (node_modules, transpiled files, etc) Less likely to change.

Reply:
same